CompanionPane/build.gradle

```diff
@@ -44,6 +44,7 @@ dependencies {
     implementation composeDependencies.activityCompose
 
     implementation microsoftDependencies.twoPaneLayout
+    implementation project(':WindowState')
```

Comment:
I think we can remove the JWM dependency for all the samples now `implementation androidxDependencies.window`.